akka-actor/src/main/scala/akka/actor/ActorCell.scala

```diff
@@ -661,11 +661,26 @@ private[akka] class ActorCell(
                or is missing an appropriate, reachable no-args constructor.
               """,
               i.getCause)
-          case x => throw ActorInitializationException(self, "exception during creation", x)
+          case x =>
+            val rootCause = rootCauseOf(x)
+            val rootCauseMessage = Option(rootCause.getMessage).getOrElse("No message available")
+            val rootStackLine = rootCause.getStackTrace.headOption.getOrElse("No stack trace available")
+            throw ActorInitializationException(
+              self,
+              s"exception during creation, root cause message:[$rootCauseMessage] at line:[${rootStackLine}]",
+              x)
```

Comment:
Hmm, so the cause was already in here. Surprising that the person reporting it in the forums did not notice that.